webpack.config.js

```diff
@@ -0,0 +1,14 @@
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const path = require('path');
+
+module.exports = {
+	entry: './formatter/index.js',
+	output: {
+		path: path.resolve(__dirname, './backend/static/js/formatter/'),
+		filename: 'index.js',
+	},
+	mode: 'development',
+	experiments: {
+		asyncWebAssembly: true
```

Comment:
What does this do?

Reply:
This enables WASM to be bundled by webpack in the first place.

The doku says:
`asyncWebAssembly`: Support the new WebAssembly according to the [updated specification](https://github.com/WebAssembly/esm-integration), it makes a WebAssembly module an async module. And it is enabled by default when [experiments.futureDefaults](https://webpack.js.org/configuration/experiments/#experimentsfuturedefaults) is set to true.